Tasks/Ananich Artem/Task_5/dz.5.py

```diff
@@ -0,0 +1,83 @@
+# 1. Реализовать функцию check_str которая получает на вход непустую строку и 
+# выдаёт информацию о количестве букв в верхнем и нижнем регистре.
+# check_str('The quick Brow Fox') -> '3 upper case, 12 lower case'
+
+# 2. Реализовать функцию is_prime которая получает на вход 
+# любое число больше нуля и выдаёт True, если число является простым, и False, если нет.
+# is_prime(787) -> True
+# is_prime(777) -> False
+
+# 3. Реализовать функцию get_ranges которая получает на вход непустой список 
+# неповторяющихся целых чисел, 
+# отсортированных по возрастанию, и которая этот список “сворачивает”.
+
+# get_ranges([0, 1, 2, 3, 4, 7, 8, 10])  ->  "0-4, 7-8, 10"
+# get_ranges([4,7,10])  -> "4, 7, 10"
+# get_ranges([2, 3, 8, 9])  -> "2-3, 8-9"
+
+#1
+def checkStr(s):
+    '''type(s) = str;
+        type(result) = str;'''
+    upper = 0
+    lower = 0
+    for i in s:
+        if (ord(i) >= 65 and ord(i) <= 96) or (ord(i) >= 1040 and ord(i) <= 1071):
+            upper += 1
+        if (ord(i) >= 97 and ord(i) <= 122) or (ord(i) >= 1072 and (ord(i) <= 1103)):
+            lower += 1
```

Comment:
whyyyy???) there are two simple functions isupper() and islower(), check them out